tests/functional/codegen/features/test_logging.py

```diff
@@ -962,7 +966,7 @@ def set_list():
     ]
 
 
-def test_logging_fails_when_input_is_too_big(assert_tx_failed, get_contract_with_gas_estimation):
+def test_logging_fails_when_input_is_too_big(tx_failed, get_contract_with_gas_estimation):
```

Comment:
most of the tests in this file can be rewritten with `with pytest.raises(...): compile_code(...)`